groclient/lib.py

```diff
@@ -421,17 +421,15 @@ def rank_series_by_source(access_token, api_host, selections_list):
         series_map[series_key][selection.get('source_id')] = selection
 
     for series_key, series_by_source_id in series_map.items():
-        try:
-            series_without_source = {
-                type_id: int(series_key.split('.')[idx])
-                for idx, type_id in enumerate(DATA_SERIES_UNIQUE_TYPES_ID)
-                if type_id != 'source_id' and series_key.split('.')[idx] != 'None'
-            }
-            source_ids = get_source_ranking(access_token,
-                                            api_host,
-                                            series_without_source)
-        except ValueError:
-            continue  # empty response
+        series_without_source = {
+            type_id: json.loads(series_key.split('.')[idx])
```

Comment:
optional/nit: maybe do `json.dumps()` instead of `str()` on line 414 to mirror the json.dumps here?

Reply:
good idea. change pushed